logger/logger.go

```diff
@@ -11,6 +11,36 @@ import (
 	"golang.org/x/net/context"
 )
 
+type Level int
+
+const (
+	OFF Level = iota
+	CRIT
+	ERROR
+	WARN
+	INFO
+	DEBUG
```

Comment:
Im not sure about these. All caps isnt very idiomatic go.
However, the Warn etc symbols are already taken by the package level logging methods.
I could break out a levels package?

Thoughts?

Reply:
You could prefix or suffix em, InfoLevel, LevelWarn